mocha.sass

```diff
@@ -263,8 +263,18 @@ body {
   right: 10px;
   font-size: 12px;
   margin: 0;
-  color: #888;
+  color: #1d1e1f;
   z-index: 1;
+  background-color: #00e600;
+  border: solid  #00e600;
+  border-radius: 10px;
+}
+
+#mocha-stats.fail {
+ @extend #mocha-stats;
```

Comment:
[Style] I don't think this `@extend` is really needed. We can achieve the same result in raw CSS, such as with CSS variables or different class names. 

Thinking in terms of semantic class names, I think it'd make sense to have a class like `.fail`, `.pass`, etc. that each define their own colors.

Answering https://github.com/mochajs/mocha/pull/4148#issuecomment-915374638: no, CSS can overcome these issues. 🙂